src/librustdoc/passes/doc_test_lints.rs

```diff
@@ -84,15 +89,15 @@ pub fn look_for_tests<'tcx>(cx: &DocContext<'tcx>, dox: &str, item: &Item) {
             return;
         }
     };
-
     let mut tests = Tests::new();
 
     find_testable_code(&dox, &mut tests, ErrorCodes::No, false, None);
 
+    // TODO: shouldn't we *not* emit warnings on private items?
```

Comment:
Won't private items already be stripped out by an earlier pass? Either way, can you add a test to be sure?

Reply:
I checked: private items emit the warnings. I'll keep them in the final test to ensure that they're not emitting the warning
 anymore.